infra/src/main/resources/db/migration/repeatable/R__03.04_layout_segment_index.sql

```diff
@@ -1,8 +1,11 @@
 drop index if exists layout.layout_segment_switch;
-create index layout_segment_switch on layout.segment(switch_id);
```

Comment:
Repeatablet on meillä lähtenyt vähän sikin sokin noiden nimeämisten kanssa. Ajattelin että voitaisiin yhtenäistää näitä hiukan (omana pullarinaan). Repeatableissakin on se ongelma että niitä ei voi helposti poistaa (olisi kyllä kikkoja, mutta..) joten jos tehdään migraatio per indeksi niin että ne tiedoston nimessä näkyy columnit ja kaikki, siitä tiedostosta ei enää pääse eroon. 

Siksi olisi ehkä parempi että pidettäisi 1 migraatiofilu per taulu ja siellä on sisällä idempotentisti luotuna kaikki sen taulun indeksit. Sitten vanhat indeksit näkyy pelkkinä drop-if-exists lauseina (kuten tässä nuo layout.segment -taulun indeksit ja voimassa olevat muodossa drop+luonti. Haittana tässä on että jos yhtä indeksiä muutetaan, kaikki taulun indeksit luodaan uudelleen, mutta indeksien luonti on aika nopeaa, enkä usko että se on mikään ongelma.